tsconfig.json

```diff
@@ -0,0 +1,24 @@
+{
+    "compilerOptions": {
+        "typeRoots": [
+            "node_modules/@types"
+        ],
+        "module": "commonjs",
+        "moduleResolution": "node",
+        "target": "es6",
+        "sourceMap": true,
+        "outDir": "js/",
+        "allowJs": true,
+        "rootDir": "src",
+        "noUnusedLocals": true,
+        "noUnusedParameters": true,
+        "strictNullChecks": true,
+        "noImplicitThis": true,
+        "noFallthroughCasesInSwitch": true
+    },
```

Comment:
Consider the following options:
`alwaysStrict`
`checkJs`
`forceConsistentCasingInFileNames`
`noEmitOnError`
`noImplicitAny`
`pretty`